src/katsdpcontroller/product_config.py

```diff
@@ -671,6 +673,11 @@ def subsampling(self) -> int:
         return self.decimation_factor if self.vlbi is None else 2 * self.decimation_factor
 
 
+class GpucbfWindowFunction(enum.Enum):
+    HANN = "hann"
+    RECT = "rect"
```

Comment:
I feel a bit iffy that 
* you've introduced an enum with a string value here, but
* [fgpu uses integer values in its version](https://github.com/ska-sa/katgpucbf/blob/main/src/katgpucbf/fgpu/output.py#L29-L34)

I mean, not the end of the world seeing as the members have the same name, right?

Reply:
Noted all around. I am definitely in support of at least updating fgpu to use the same values as here. Agreed on the usefulness of `parse_enum`.